resources/views/forum/topics/_post_info.blade.php

```diff
@@ -40,6 +40,12 @@ class="forum-post-info__row forum-post-info__row--username js-usercard"
             data-user-id="{{$user->user_id}}"
             href="{{ route("users.show", $user) }}"
         >{{ $user->username }}</a>
+
+        @if ($user->title())
```

Comment:
Although usually fine, conditional test using plain string in php should be avoided. Use `isset` check.

Reply:
seems like ``isset`` won't work in this context, will go for a regular check instead lol